lib/web_ui/lib/src/engine/canvaskit/embedded_views.dart

```diff
@@ -93,47 +100,63 @@ class HtmlViewEmbedder {
     _frameSize = size;
   }
 
-  List<CkCanvas> getCurrentCanvases() {
+  /// Returns a list of canvases which will be overlaid on top of the "base"
+  /// canvas after a platform view is composited into the scene.
+  ///
+  /// The engine asks for the overlay canvases immediately before the paint
+  /// phase, after the preroll phase. Note that in the preroll phase we must
```

Comment:
nit: our style guide doesn't like "Note that" https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose

Reply:
Done